ReactNativeClient/lib/services/InteropService_Importer_Md.js

```diff
@@ -54,21 +55,51 @@ class InteropService_Importer_Md extends InteropService_Importer_Base {
 		}
 	}
 
+	/**
+	 * Parse text for links, attempt to find local file, if found create Joplin resource
+	 * and update link accordingly.
+	 */
+	async importLocalImages(filePath, md) {
+		let updated = md;
+		const imageLinks = lo.uniq(extractImageUrls(md));
```

Comment:
Please use `ArrayUtils.unique` as for now I'm trying to limit the use of lodash.

Reply:
👌 , done